packages/ListBox/src/components/Filter/Filter.js

```diff
@@ -87,10 +87,19 @@ const Filter = React.forwardRef((props, ref) => {
   };
 
   React.useEffect(() => {
+    let id = null;
+
     if (!isOpen) {
-      reset();
```

Comment:
What happened to `reset()`?  Can it be removed?

Reply:
I don't fully remember why I removed it, but I refactored it to avoid one of those errors in a test about wrapping the component in an `act()` and during that, I realized that the reset was not longer necessary with all the refactor that happened on the list box and the keyboard.